ckanext/data_qld_theme/templates/user/dashboard.html

```diff
@@ -0,0 +1,18 @@
+{% ckan_extends %}
+
+{% block page_header %}
+<header class="module-content page-header hug">
+  <div class="content_action">
+    {% link_for _('Edit settings'), named_route='user.edit', id=user_dict.get('name', ''), class_='btn btn-default', icon='cog' %}
+  </div>
+  <ul class="nav nav-tabs">
+    {{ h.build_nav_icon('dashboard.index', _('News feed')) }}
+    {{ h.build_nav_icon('dashboard.datasets', _('My Datasets')) }}
+    {{ h.build_nav_icon('dashboard.organizations', _('My Organizations')) }}
+    {{ h.build_nav_icon('dashboard.groups', _('My Groups')) }}
+    {% if h.check_access('has_user_permission_for_some_org', {'permission': 'create_dataset'}) %}
```

Comment:
I'll accept, tho we should look at improving the ckan core to allow additions without complete override of the core template. i.e we get injected into the core
{% block page_header_nav %}
(% endblock %}
and then we can add items before and after the nav items.

Reply:
Agreed - this is a pain point with the Jinja template inheritance model. It would be great to be able to add dashboard nav items piecemeal. I guess it's a battle for another day.